README.md

```diff
@@ -27,99 +27,32 @@ The following platforms are officially supported (tested):
 
 ### Installation
 
-You can install the Frequenz Electricity Trading API client via pip. Replace `VERSION` with the specific version you wish to install.
+We assume you are on a system with Python available. If that is not the case,
```

Comment:
As a reader of it feels a bit odd to only find "Installation" under the point "usage" but not a single mentioning of how to actually use it?

Reply:
Fair enough, I actually think the Usage section can be removed here